.packit.yaml

```diff
@@ -16,8 +16,10 @@ actions:
 jobs:
   - job: copr_build
     trigger: pull_request
+    notifications:
+      failure_comment:
+        message: "Ephemeral COPR build failed. @lsm5 please check."
```

Comment:
fine for a draft, but please remove lsm for final.

Reply:
@TomSweeneyRedHat that's from https://github.com/containers/podman/pull/20061 . It's intentional.